reference/tools/data-migration/faq.md

```diff
@@ -50,3 +50,15 @@ DM 会尝试将包含多个 DDL 变更操作的单条语句拆分成只包含一
 5. 使用 Ansible [启动整个 DM 集群](/how-to/deploy/data-migration-with-ansible.md#第-9-步部署-dm-集群)。
 
 6. 以新的任务名重启数据同步任务，或设置 `remove-meta` 为 `true` 且 `task-mode` 为 `all`。
+
+## 设置了 **online-ddl-sheme: gh-ost** , 但是 DM 还是出现关于 gh-ost 相关的表的错误
+
+> [unit=Sync] ["error information"="{\"msg\":\"[code=36046:class=sync-unit:scope=internal:level=high] online ddls on ghost table `xxx`.`_xxxx_gho`\\ngithub.com/pingcap/dm/pkg/terror.(*Error).Generate ......
+
+由于 DM 是在最后 rename ghost_table to origin table 的步骤会把内存的 ddl 信息读出，并且还原为 origin table 的 ddl 。而内存中的 ddl 信息是在 alter ghost_table 的时候进行[处理](/reference/tools/data-migration/features/online-ddl-scheme.md#dm-对于-online-ddl-scheme-gh-ost-的处理) ，把 ghost_table ddl 的信息进行记录。或者是在重启 dm-woker 启动 task 的时候，从 dm_meta.{task_name}_onlineddl 中读取出来。因此，如果在增量同步过程中，指定的 Pos 跳过了 alter ghost_table 的 ddl ，但是该 pos 仍在 gh-ost 的 online-ddl 的过程。就会因为 ghost_table 没有正确写入到内存以及 dm_meta.{task_name}_onlineddl ，而导致该问题。  
```

Comment:
Please check whether this is what you mean😀 By separating the original answer into several paragraphs, we can make it easier for users to read. 
Also, please use the blacktick ``` ` ``` to enclose SQL statements and variables.
```suggestion
出现上述错误可能由于以下原因：

DM 在最后 `rename ghost_table to origin table` 的步骤会把内存的 DDL 信息读出，并且还原为 origin table 的 DDL 。而内存中的 DDL 信息是在 `alter ghost_table` 的时候进行[处理](/reference/tools/data-migration/features/online-ddl-scheme.md#dm-对于-online-ddl-scheme-gh-ost-的处理) 并记录 ghost_table DDL 的信息，或者重启 dm-worker 启动 task 时，从 `dm_meta.{task_name}_onlineddl` 中读取出来。

因此，如果在增量同步过程中，指定的 Pos 跳过了 `alter ghost_table` 的 DDL ，而该 pos 仍在 gh-ost 的 online-ddl 过程中，就会因为 ghost_table 没有正确写入到内存以及 `dm_meta.{task_name}_onlineddl`，而导致该问题。  
```